mansur_adam_ndvi.py

```diff
@@ -0,0 +1,892 @@
+
+
+# In[12]:
+
+
+# Ignore this cell for the autograding tests
+
+
+# # Question 1 (10 points)
+# 
+# Imagine that you are planning NEON’s upcoming flight season to capture remote sensing data in these locations and want to ensure that you fly the area when the vegetation is the most green.
+# 
+# When would you recommend the flights take place for each site? 
+# 
+# Answer the question in 2-3 sentences in the Markdown cell below.
+
+# Healthy, green vegetation will show a moderate to high NDVI. The highest NDVI (and
+# therefore greenest vegeation) occurred at the SJER site between March and mid-April
+# and at the HARV site between mid-May and late September, so those would be the best
+# windows for scheduling flights.
+
+# # Question 2 (10 points)
+# 
+# How could you modify your workflow to look at vegetation changes over time in each site? 
+# 
+# Answer the question in 2-3 sentences in the Markdown cell below.
+
+# One approach would be to compare results from periods of high and low NDVI. For SJER,
+# that would mean scheduling a flight for sometime between July and December in addition
+# to the high NDVI flight above. For HARV, the low NDVI flight would be scheduled for
+# March to April (and possibily earlier since data from January and February are not
+# available but NDVI at this site is trending down in December).
+
+# # Do not edit this cell! (10 points)
+# 
+# The notebook includes:
+# * additional Markdown cells throughout the notebook to describe: 
+#     * the data that you used - and where it is from
+#     * how data are being processing
+#     * how the code is optimized to run fast and be more concise
+
+# # Do not edit this cell! (20 points)
+# 
+# The notebook will also be checked for overall clean code requirements as specified at the **top** of this notebook. Some of these requirements include (review the top cells for more specifics): 
+# 
+# * Notebook begins at cell [1] and runs on any machine in its entirety.
+# * PEP 8 format is applied throughout (including lengths of comment and code lines).
+# * No additional code or imports in the notebook that is not needed for the workflow.
+# * Notebook is fully reproducible. This means:
+#    * reproducible paths using the os module.
+#    * data downloaded using code in the notebook.
+#    * all imports at top of notebook.
+
+# ## BONUS - Export a  .CSV File to Share (10 points possible)
+# 
+# This is optional - if you export a **.csv** file with the columns specified above: Site, Date and NDVI Value you can get an additional 10 points.
+# 
+# * FULL CREDIT: File exists in csv format and contains the columns specified.
+# We will check your github repo for this file!
+# 
+
+# In[13]:
+
+
+# Export CSV with mean NDVI to outputs folder
+mean_ndvi_all.to_csv(os.path.join("ndvi-automation", "outputs", "mean_ndvi.csv"),
+                     header=["Site", "NDVI Value"],
+                     index_label="Date")
+
+
+# In[ ]:
+
+
```

Comment:
@adamancer Awesome job Adam! I feel lucky to have partnered with you on this assignment because I learned so much from your approach!